tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -38,31 +38,63 @@ def main() -> int:
     # Process initialization/setup must be done early. NOTE: do NOT log before this line!
     process_init(build_args)
 
-    # Return process exit code (or raise, which exits with a code of `1`)
-    return do_build(build_args, skip_completed_steps=cli_args.test_resume)
+    # List of available commands for the builder
+    commands = {
+        "release": [do_the_release],
+        "replicate": [do_sync_artifact_to_replica_s3_bucket],
+        "sync-release": [
+            do_sync_release_file_to_replica_s3_bucket,
+        ],
+        "build": [
+            do_prebuild_set_defaults,
+            do_prebuild_checks,
+            do_build_soma,
+            do_validate_soma,
+            do_create_reports,
+            do_data_copy,
+            do_report_copy,
+            do_log_copy,
+        ],
+        "cleanup": [
+            do_old_release_cleanup,
+        ],
+        "full-build": [
```

Comment:
Sure, I can rename that. This is unused anyway.

Reply:
delete it then?

Reply:
It is used by the default behavior. This was Bruce's request in case you want to build manually. I agree with you that it would make more sense to only do a "local" build, so I'll comply.

Reply:
I'm fine changing it!